server/base/src/main/java/org/apache/accumulo/server/util/MetadataTableUtil.java

```diff
@@ -183,6 +183,7 @@ public static void updateTabletDataFile(long tid, KeyExtent extent,
     tablet.putTime(time);
     estSizes.forEach(tablet::putFile);
 
+    // TODO check to see if we need to delete.. putBulkFile does an insert
```

Comment:
Should resolve this before merge, or create follow-on task.

Reply:
I think this is OK since it is only called by the 2 bulk import methods.  And each has a corresponding CleanupBulkImport FATE operations which calls```MetadataTableUtil.removeBulkLoadEntries()```